packages/core/src/toggle/toggle.element.ts

```diff
@@ -35,6 +35,26 @@ import { styles } from './toggle.element.css.js';
  * @cssprop --toggle-speed
  */
 export class CdsToggle extends CdsInternalControlInline {
+  @property({ type: String })
+  cdsMotion = 'on';
+
+  updated(props: Map<string, any>) {
+    super.updated(props);
+
+    // this section fixes an issue with toggles animating when the page loads
+    // this work has to be deferred and executed outside the current update loop
+    // note that setting cdsMotion to 'off' will turn off all animation
+    // https://github.com/vmware/clarity/issues/5880
+    if (props.has('cdsMotion') && this.cdsMotion === 'on') {
+      const defer = setTimeout(() => {
+        if (this.cdsMotion === 'on') {
+          this.cdsMotion = 'ready';
+        }
+        clearTimeout(defer);
+      }, 10);
+    }
+  }
+
```

Comment:
I think we could remove the need for the timeout by hooking into the first updated lifecycle. We can have the animation set to 0 until the ready state is set.

```scss
:host {
  --toggle-speed: #{$cds-global-animation-duration-quicker};
}

:host([cds-motion='on']) {
  --toggle-speed: 0;
}
```

```typescript
  @property({ type: String, reflect: true }) cdsMotion = 'on';

  firstUpdated(props: Map<string, any>) {
    super.firstUpdated(props);

    if (this.cdsMotion === 'on') {
      this.setAttribute('cds-motion', 'ready');
    }
  }
```